lib/spack/spack/util/web.py

```diff
@@ -390,7 +607,7 @@ def get_checksums_for_versions(
             *spack.cmd.elide_list(
                 ["{0:{1}}  {2}".format(str(v), max_len, url_dict[v])
                  for v in sorted_versions]))
-    print()
+    tty.msg('')
```

Comment:
why?  print() adds an empty line, while `tty.msg()` prints an arrow.

Reply:
That's a think-o on my part.  Will fix.